hapi/src/services/eden-historic-delegates.service.js

```diff
@@ -1,11 +1,17 @@
+const moment = require('moment')
+
 const {
   edenDelegatesGql,
   edenHistoricElectionGql,
-  edenElectionGql
+  edenElectionGql,
+  edenTransactionGql
 } = require('../gql')
 const { servicesConstant } = require('../constants')
 const { sleepUtil, eosUtil, axiosUtil } = require('../utils')
-const { hyperionConfig, edenConfig } = require('../config')
+const { hyperionConfig, edenConfig, eosConfig } = require('../config')
+
+let LASTEST_RATE_DATE_CONSULTED = null
+let LASTEST_RATE_DATA_CONSULTED = null
```

Comment:
Why are these variables declared here?

Reply:
To avoid consulting the same date several times